sky/shell/platform/ios/framework/Headers/FlutterAppDelegate.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERAPPDELEGATE_H_
```

Comment:
These are only needed for c++ includes, btw, since Obj-C uses #import instead of #include to de-dupes headers automagically.